deploy/roles/role.yaml

```diff
@@ -48,6 +48,7 @@ rules:
 - apiGroups:
   - integreatly.org
   resources:
+  - grafanas/status
```

Comment:
I didn't realise status was a subresource.
Is this fixing an unrelated bug or errors in logs?

Reply:
When this is not present it's causing an error on OS3

Reply:
it was fixing permission error in the logs